R/namespace.R

```diff
@@ -229,6 +229,14 @@ roxy_tag_parse.roxy_tag_importClassesFrom <- function(x) {
 }
 #' @export
 roxy_tag_ns.roxy_tag_importClassesFrom <- function(x, block, env, import_only = FALSE) {
+  pkg <- x$val[1L]
+  if (requireNamespace(pkg, quietly = TRUE)) {
+    importing <- x$val[-1L]
+    unknown_idx <- !importing %in% getNamespaceExports(pkg)
+    if (any(unknown_idx)) {
+      cli::cli_abort("Unknown {cli::qty(sum(unknown_idx))} export{?s} in {.code @importFrom {pkg}}: {.code {importing[unknown_idx]}}")
```

Comment:
Sorry, I remembered that this should use `roxy_warn_tag()` as that will also give information about where the tag was found.

Reply:
Hmm I had stayed away from that since `warn_roxy_tag()` will still proceed to write the invalid imports in the NAMESPACE. Switched to `warn_roxy_tag()` for now but is that the best behavior?